### PR TITLE
added r_ and c_ to jax.numpy

### LINKED ISSUE #21765: r_

### LINKED ISSUE #21766: c_

### PR DESCRIPTION
**Pull request by ShreyanshBardia**
closes #21766 
closes #21765

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by ShreyanshBardia** [comment]
Hi @AnnaTz, this PR is ready for review

File: ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_indexing.py [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
 import numpy as np
-from jax.numpy import tril, triu
+from jax.numpy import tril, triu, r_, c_
 
 
 # local
 import ivy_tests.test_ivy.helpers as helpers
-from ivy_tests.test_ivy.helpers import handle_frontend_test
+from ivy_tests.test_ivy.helpers import handle_frontend_test, update_backend
+from ivy_tests.test_ivy.test_frontends.test_numpy.test_indexing_routines.test_inserting_data_into_arrays import (
```
Comment (AnnaTz) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_indexing.py:10]: Please resolve the lint error detected by our CI checks. Seems to be caused by this line being too long.
Other than that, looks good. Thanks!

**Review by AnnaTz — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_indexing.py (hunk 1 above)
Comment (ShreyanshBardia) [reply ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_indexing.py]: Done, can you check now @AnnaTz

**Review by AnnaTz — APPROVED** [review]
Great, thanks!

**Pull request merged**